db/seeds.rb

```diff
@@ -455,6 +455,17 @@
       link: 'https://balticruby.org/',
       img: 'https://thumb.tildacdn.com/tild6166-3665-4835-b736-363836373334/-/format/webp/logovector.png.webp'
     },
+    {
+      slug: 'euruko-2025',
+      name: 'Euruko 2025',
+      event_date: 'Viana do Castelo, Portugal',
+      location: 'September 18-19, 2025',
+      description: 'The largest and longest running annual Ruby conference in Europe, since 2003, that brings you all the good news about Ruby and its friends. Every year it takes place in a different European city, the location being chosen by participants.',
+      latitude: 41.6936837,
+      longitude: -8.8300416,
+      link: 'https://2025.euruko.org/',
+      img: 'https://2025.euruko.org/images/logo.svg'
```

Comment:
Can you please put the logo directly in repository, similar to how it was done in the example PR? https://github.com/rubyeurope/rubyeurope.com/pull/1/files

We are trying to be independent from availability of other servers